2024/12.md

```diff
@@ -115,6 +115,8 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     |:-----:|:-------:|-------|-----------|
     | 3     | 5m      | [Intl.DurationFormat](https://github.com/tc39/proposal-intl-duration-format) Correct spec bug resulting in fraction digits and rounding mode options being ignored. ([PR](https://github.com/tc39/proposal-intl-duration-format/pull/203/)) | Ben Allen |
     | 2.7   | 15m     | [`Error.isError](https://github.com/tc39/proposal-is-error/issues/7) to stage 3? | Jordan Harband |
+    | 1     | 30m   | [Immutable ArrayBuffer](https://github.com/tc39/proposal-immutable-arraybuffer) to stage 2? | Mark Miller |
+    | 0     | 60m   | Stabilize (link TBD) to stage 1? | Mark Miller |
```

Comment:
😲 what is this Stabilize thing?

Is this some kind of built-in interning, possibly an alternative to Records & Tuples value semantics?
https://github.com/tc39/proposal-record-tuple/issues/387

cc @acutmore any idea?

Reply:
that is indeed the link